api/lib/chat/chat_bloc.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:api/chat/chat_model.dart';
```

Comment:
I think we should keep this in session bloc, as chats will exist as long as a session exists. Making a separate connection doesn't make sense. Thoughts?

Reply:
Websocket for drawing and chat is different, separation of concerns.

Reply:
no the connection is unique, we don't establish separate connection for them

Reply:
on FE repositories can be different but on BE chat is a part of session

Reply:
makes sense, will change.